src/scripts/main.js

```diff
@@ -358,3 +358,24 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+const elemTable = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in your HTML. If it doesn't, `elemTable` will be `null`, and attempting to append rows to it will result in an error. Consider adding a check to ensure `elemTable` is not `null` before proceeding with the loop.